### DISCUSSION TITLE
tabella ansc_3

### DISCUSSION THREAD

**Discussion by annamaraspin** [italia/ansc#1190] [Supporto servizi cooperativi]
Nella versione 1.35 per il caso 459200 la colonna idtipocontenuto è valorizzato a 3 (trascrizione)
Non dovrebbe essere 2 (annotazione)?

**Comment by acolosimosogei** [comment]
Buongiorno, 
confermo quanto da lei riportato nella discussion, di conseguenza abbiamo apportato la modifica correttiva sulla tabella ansc_3.
Procedo a chiudere la discussion, ricordo che potrà essere riaperta in futuro.

Tante grazie per la segnalazione,
Assistenza ANSC
